iroh-net/src/magicsock/derp_actor.rs

```diff
@@ -233,10 +235,10 @@ impl ActiveDerp {
                             src: source,
                             buf: data,
                         };
-                        self.msg_sender
-                            .send(ActorMessage::ReceiveDerp(res))
-                            .await
-                            .ok();
+                        if let Err(err) = self.msg_sender.try_send(ActorMessage::ReceiveDerp(res)) {
```

Comment:
I have seen this happen a lot in shutdown, so I think this seems like a good idea, but also means we are not slowing down here, when the magicsock is overloaded.